create_countrycities.ipynb

```diff
@@ -88,7 +88,7 @@
    "outputs": [],
```

Comment:
<p>Where did this list come from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wri/SCL-cities/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>